src/main/kotlin/no/nav/syfo/personstatus/application/OppfolgingstilfelleService.kt

```diff
@@ -36,9 +32,24 @@ class OppfolgingstilfelleService(
                     oppfolgingstilfelle = newPersonOppfolgingsTilfelle,
                 )
             }
+            if (existingPerson.veilederIdent != null) {
+                checkTildeltVeileder(existingPerson.veilederIdent)
+            }
         }
     }
 
+    private suspend fun createPersonWithNameAndFodselsdato(personStatus: PersonOversiktStatus) {
+        pdlClient.getPerson(PersonIdent(personStatus.fnr))
+            .map {
+                val editedPersonStatues =
```

Comment:
Ser det henger igjen en typo fra før her 
```suggestion
                val editedPersonStatuses =
```